### PR TITLE
Add priority to AmlCheck response

### PR DESCRIPTION
**Pull request by mikerebilly**
## Summary
Add `priority` to AmlCheck API response schema.

## Links
N/A

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/AML/Settings/AmlSettingsPriority.yaml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 type: object
-description: Priority settings.
+description: |-
+  Priority level settings for AML source matches. \
+  `p0` is critical priority level, and `p3` is low priority level. \
+  All source types in all hits are considered, and the one with the highest priority level is set in the resulting AML check.
```
Comment (noon-dawg) [inline openapi/components/schemas/AML/Settings/AmlSettingsPriority.yaml:5]: suggestion: minor suggestion
```suggestion
  All source types are assessed when AML matches are found. 
  Sources with the highest priority levels are added to the AML check result.
```

**Review by colinbird — DISMISSED** [review]

File: openapi/components/schemas/AML/Settings/AmlSettingsPriority.yaml (hunk 1 above)
Comment (mikerebilly) [reply openapi/components/schemas/AML/Settings/AmlSettingsPriority.yaml:5]: This is incorrect though. The highest priority overrides the others.

Comment (mikerebilly) [reply openapi/components/schemas/AML/Settings/AmlSettingsPriority.yaml]: I've removed it from this area and changed description of the field in AmlCheck.

**Comment by mikerebilly** [comment]
php please review

**Comment by mikerebilly** [comment]
js please re-review

File: openapi/components/schemas/AML/Settings/AmlSettingsPriority.yaml [hunk 2]
```diff
@@ -1,5 +1,7 @@
 type: object
-description: Priority settings.
+description: |-
+  Priority level settings for AML source matches. \
```
Comment (nanobash) [inline openapi/components/schemas/AML/Settings/AmlSettingsPriority.yaml:3]: Is `\` really needed here?

```suggestion
  Priority level settings for AML source matches.
```

Comment (mikerebilly) [reply openapi/components/schemas/AML/Settings/AmlSettingsPriority.yaml:3]: That's meant to be there, yes.

Comment (noon-dawg) [reply openapi/components/schemas/AML/Settings/AmlSettingsPriority.yaml:3]: It adds a new line

**Review by noon-dawg — APPROVED** [review]

**Review by colinbird — APPROVED** [review]

**Pull request merged**